### PR TITLE
Add columns to input

### PR DESCRIPTION
**Pull request by cherta**
This PR adds columns to the text input. The goal is to have an extra field on the input pop up that reads the same as the textarea columns.

### CODE REVIEW

File: lib/simditor.js [hunk 1]
```diff
@@ -503,7 +498,8 @@ Formatter = (function(superClass) {
       h1: ['margin-left', 'text-align'],
       h2: ['margin-left', 'text-align'],
       h3: ['margin-left', 'text-align'],
-      h4: ['margin-left', 'text-align']
+      h4: ['margin-left', 'text-align'],
+      input: ['width']
```
Comment (jubar) [inline lib/simditor.js:502]: I think this is not necessary now since we are resolving the width of the input via *style* attr.

Comment (cherta) [reply lib/simditor.js:502]: Yes it is, I tested and it doesn't work if you don't specify the actual prop of the style

File: src/buttons/input.coffee [hunk 2]
```diff
@@ -94,12 +94,20 @@ class InputPopover extends Popover
             </select>
           </td>
         </tr>
+        <tr>
+          <td class="field-name">
+            <label>Columns:</label>
+          </td>
+          <td>
+            <input class="simditor-input-width" type="number" min="0" />
```
Comment (jubar) [inline src/buttons/input.coffee:102]: Min not should be 1 like in the textarea config?

File: src/formatter.coffee [hunk 3]
```diff
@@ -41,6 +41,7 @@ class Formatter extends SimpleModule
       h2: ['margin-left', 'text-align']
       h3: ['margin-left', 'text-align']
       h4: ['margin-left', 'text-align']
+      input: ['width']
```
Comment (jubar) [inline src/formatter.coffee:44]: I think this is not necessary now since we are resolving the width of the input via style attr.

**Review by jubar — APPROVED** [review]

File: src/buttons/input.coffee (hunk 2 above)
Comment (cherta) [reply src/buttons/input.coffee:102]: If it's 0 then it's falsy and removes the attribute as a whole

File: src/formatter.coffee (hunk 3 above)
Comment (cherta) [reply src/formatter.coffee:44]: Same as above

Comment (jubar) [reply src/formatter.coffee:44]: you are right, I didn't notice that you are specify style attributes.

**Pull request merged**